src/Raven.Studio/typescript/commands/commandBase.ts

```diff
@@ -3,8 +3,8 @@
 import messagePublisher = require("common/messagePublisher");
 import appUrl = require("common/appUrl");
 import protractedCommandsDetector = require("common/notifications/protractedCommandsDetector");
-import twoFactorHelper from "common/twoFactorHelper";
-import database from "models/resources/database";
+import twoFactorHelper = require("common/twoFactorHelper");
```

Comment:
Now ES Modules is a standard, so it's better to do the opposite, i.e. replace CommonJS imports with ECMAScript ones

Reply:
discussed - the current approach requires much less to change

Reply:
(handled in PM)